src/Aquifer.API/Endpoints/Resources/ResourceReferences/Delete/Endpoint.cs

```diff
@@ -0,0 +1,48 @@
+using Aquifer.API.Common;
+using Aquifer.Data;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Resources.ResourceReferences.Delete;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request>
+{
+    public override void Configure()
+    {
+        Delete("/resources/resource-references");
+        Permissions(PermissionName.EditContentResourceReferences);
+    }
+
+    public override async Task HandleAsync(Request request, CancellationToken ct)
+    {
+        var resourceContent = await dbContext.ResourceContents
+            .Include(rc => rc.Resource)
+            .ThenInclude(r => r.AssociatedResourceChildren)
+            .SingleOrDefaultAsync(rc => rc.Id == request.ResourceContentId, ct);
+
+        if (resourceContent == null)
+        {
+            await SendNotFoundAsync(ct);
+            return;
+        }
+
+        var referenceResource = await dbContext.Resources.FindAsync([request.ReferenceResourceId, ct], ct);
```

Comment:
Don't think you meant to pass `ct` twice here.

Reply:
Also, what's the reason for pulling this? It doesn't seem like we do anything with it. If `associatedResource` below is null, we could throw 404 there.